tutorial/Lab1/README.md

```diff
@@ -0,0 +1,200 @@
+# Lab 1. I just want to deploy!
+
+Let's investigate how Helm can help us focus on other things by letting a chart do the work for us. We'll first deploy an application to a Kubernetes cluster by using `kubectl` and then show how we can offload the work to a chart by deploying the same app with Helm.
+
+The application is the [Guestbook App](https://github.com/IBM/guestbook), which is a sample multi-tier web application.
+
+# Deploy the application using `kubectl`
+
+In this part of the lab, we will deploy the application using the Kubernetes client `kubectl`. We will use [Version 1](https://github.com/IBM/guestbook/tree/master/v1) of the app for deploying here. Clone the [Guestbook App](https://github.com/IBM/guestbook) repo to get the files: 
+```$ git clone https://github.com/IBM/guestbook.git``` .
+
+1. Use the configuration files in the cloned Git repository to deploy the containers and create services for them by using the following commands:
+
+   ```console
+   $ cd guestbook/v1
+
+   $ kubectl create -f redis-master-deployment.yaml
+   deployment.apps/redis-master created
+
+   $ kubectl create -f redis-master-service.yaml
+   service/redis-master created
+
+   $ kubectl create -f redis-slave-deployment.yaml
+   deployment.apps/redis-slave created
+
+   $ kubectl create -f redis-slave-service.yaml
+   service/redis-slave created
+
+   $ kubectl create -f guestbook-deployment.yaml
+   deployment.apps/guestbook-v1 created
+
+   $ kubectl create -f guestbook-service.yaml
+   service/guestbook created
+   ```
+   Refer to the [guestbook README](https://github.com/IBM/guestbook) for more details.
+ 
+2. View the guestbook:
+
+You can now play with the guestbook that you just created by opening it in a browser (it might take a few moments for the guestbook to come up).
+
+ * **Local Host:**
+    If you are running Kubernetes locally, view the guestbook by navigating to `http://localhost:3000` in your browser.
+
+ * **Remote Host:**
+    1. To view the guestbook on a remote host, locate the external IP of the load balancer in the **IP** column of the `kubectl get services` output. 
+
+    ```console
+    $ kubectl get services
+    NAME           TYPE           CLUSTER-IP       EXTERNAL-IP   PORT(S)          AGE
+    guestbook      LoadBalancer   172.21.252.107   50.23.5.136   3000:31838/TCP   14m
+    redis-master   ClusterIP      172.21.97.222    <none>        6379/TCP         14m
+    redis-slave    ClusterIP      172.21.43.70     <none>        6379/TCP         14m
+    .........
+    ```
+
+    Note: If no external IP is assigned, then you can get the external IP with the following command:
+
+    ```console
+    $ kubectl get nodes -o wide
+NAME           STATUS    ROLES     AGE       VERSION        EXTERNAL-IP      OS-IMAGE             KERNEL-VERSION      CONTAINER-RUNTIME
+10.47.122.98   Ready     <none>    1h        v1.10.11+IKS   173.193.92.112   Ubuntu 16.04.5 LTS   4.4.0-141-generic   docker://18.6.1
+    ``` 
```

Comment:
I think you need a blank line after the \`\`\` above otherwise it doesn't render correctly

Reply:
done